src/deviceModeInit.js

```diff
@@ -72,7 +72,6 @@ let _rudderTracking = (function () {
       rs$.ajax({
         url: webhookUrl,
         method: 'GET',
-        contentType: 'application/json',
         timeout: 2000, // 2 seconds timeout
```

Comment:
Let's also fix the console statement on Ln 88 to convert that into a warning/error which also indicates the fallback option we'd take.